### DISCUSSION TITLE
datasets/288/ABACUS-ncpp-efficiency-precision-testdata

### DISCUSSION THREAD

**Discussion by giscus** [deepmodeling/AIS-Square#237] [Announcements]
# datasets/288/ABACUS-ncpp-efficiency-precision-testdata

https://www.aissquare.com/datasets/detail?pageType=datasets&name=ABACUS-ncpp-efficiency-precision-testdata&id=288

**Comment by kirk0830** [comment]
Update: 
- correct the delta values of X2O5 and XO3 for all elements, correct the delta values of of X2O for 1st period elements.
- delete the delta data for PSlibrary 0.3.1 of Lanthanides
